### PR TITLE
feat(content): Add bimbibap to the game

### PR DESCRIPTION
**Pull request by RobbieNeko**
### Required

- [x] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [x] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [x] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [x] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change

The food you could make with rice did not inspire joy. Bimbibap, on the other hand, inspires much joy.

In other words: There were relatively few rice dishes available in-game, and I saw the opportunity to add one of my personal favorites.

## Describe the solution

Adds bimbibap, a korean dish. Also adds gochujang, a new spice that is integral to bimbibap

## Describe alternatives you've considered

- Less big recipe for the bimbibap

Fair, but it's based on an IRL recipe I found online and besides, we can always do with tasty culinary flexes.

- Some other rice dish
- Implosion

## Testing

Loads on my save

## Additional context

Unsure of what groups / cooking requirements, if any, to add these to. Very much so open to input.

If the comestibles test screams at the bimbibap, I shall scream into its ears in kind.

### CODE REVIEW

**Review by RoyalFox2140 — APPROVED** [review]
Good to merge when calorie tests pass.
